pkg/disttask/framework/taskexecutor/task_executor_test.go

```diff
@@ -396,14 +361,14 @@ func TestTaskExecutor(t *testing.T) {
 	// no subtask to run, should exit the loop after some time.
 	checkIntervalBak := checkInterval
 	maxIntervalBak := maxCheckInterval
+	checkBalanceSubtaskIntervalBak := checkBalanceSubtaskInterval
 	defer func() {
 		checkInterval = checkIntervalBak
 		maxCheckInterval = maxIntervalBak
+		checkBalanceSubtaskInterval = checkBalanceSubtaskIntervalBak
 	}()
-	maxCheckInterval, checkInterval = time.Millisecond, time.Millisecond
+	maxCheckInterval, checkInterval, checkBalanceSubtaskInterval = time.Millisecond, time.Millisecond, time.Millisecond
```

Comment:
why change this?

Reply:
revert it 2d586c05f513d698c63f4349410fcd3202686e1d